net/speedtest/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+version             1.1.1.84
+revision            0
+
+name                speedtest
+categories          net
+platforms           darwin
+supported_archs     noarch
+maintainers         {@linuxgemini linuxgemini.space:macportspackaging} openmaintainer
+
+homepage            https://speedtest.net/apps/cli
+license             restrictive nomirror noncommercial
+
+description         official cli for speedtest.net
+long_description    command line interface for testing internet bandwidth using speedtest.net
+
+master_sites        https://install.speedtest.net/app/cli/
+distname            ookla-speedtest-${version}-macosx-x86_64
+extract.suffix      .tgz
+
+# ookla already ships this as an universal binary
+universal_variant   no
+supported_archs     x86_64 arm64
+
+checksums           rmd160  c16334261d6858b21ef08ebb4886ced3c303420e \
+                    sha256  153f76e4e502a2bb470c7f7c3b4b8afaa50eb98bb11e86216fcb1a8b92676e94 \
+                    size    1019149
+
+use_configure       no
+build {}
+
+destroot {
+    set installdir ${prefix}/bin
+
+    xinstall -d ${destroot}${installdir}
+    xinstall -m 755 -W ${workpath} \
```

Comment:
Since evidently this distfile does not extract with an enclosing directory, we prefer for the port to ask MacPorts to create that enclosing directory by using `extract.mkdir yes`. Then all references to `${workpath}` in the Portfile will need to be changed to `${worksrcpath}` as per usual.

Reply:
Resolved in b5c45453f88a930d37fc00cb64eaf0283cf5e57c.